client/src/hooks/subgraph/useEpochTimestampHappenedIn.ts

```diff
@@ -21,15 +23,24 @@ export default function useEpochTimestampHappenedIn(
   timestampSeconds: string,
 ): UseQueryResult<number> {
   const { subgraphAddress } = env;
+  const { data: currentEpoch } = useCurrentEpoch();
+  const { data: isDecisionWindowOpen } = useIsDecisionWindowOpen();
   const timestampSecondsNumber = parseInt(timestampSeconds, 10);
 
   return useQuery<GetEpochTimestampHappenedInQuery, any, number, any>({
-    enabled: !!timestampSecondsNumber,
+    enabled: window.Cypress
+      ? currentEpoch !== undefined && isDecisionWindowOpen !== undefined
+      : !!timestampSecondsNumber,
```

Comment:
Why?

Reply:
Timestamp in the App is equal to epoch 0.